zulipterminal/ui_tools/views.py

```diff
@@ -1610,13 +1619,13 @@ def __init__(
 
             # slice_index = Number of labels before topic links + 1 newline
             #               + 1 'Topic Links' category label.
-            slice_index = len(msg_info[0][1]) + 2
+            #               + 2 for Viewing Actions category label and its newline
+            slice_index = len(msg_info[0][1]) + len(msg_info[1][1]) + 2 + 2
```

Comment:
Why 2+2 here, but 4 above?

Reply:
It was a blunder by my side, has been fixed